src/actions/secrets.js

```diff
@@ -50,6 +81,7 @@ export function deleteSecret(secrets) {
     dispatch({ type: 'SECRET_DELETE_REQUEST' });
     const deletePromises = secrets.map(secret => {
       const { name, namespace } = secret;
+      unpatchServiceAccount(name, namespace);
```

Comment:
should we be waiting for this before deleting the credential or is it safe to do both in parallel?

Reply:
It's safe to do both these in parallel